### PR TITLE
fixes a way to kill colossus with no work

### PR DESCRIPTION
**Pull request by 1393F**
## About The Pull Request
like the old body chair method, but better

grab a dead goldgrub, run up to a colossus, and start shooting! It will block the colossi's bolts as they're direct hit or whatever, while you can shoot over the grub! Walk in a straight line firing your KA with the grub pulled behind you for a quick easy kill

this PR prevents that by making the bolts pierce the grub, and hit you!
## Why It's Good For The Game
no more easy colossus kills via goldgrub
## Changelog
:cl:
fix: you can nolonger use a goldgrub as a shield to kill colossus
/:cl:

### CODE REVIEW

**Comment by the-orange-cow** [comment]
Do live goldgrubs still reflect the bullets?  imo live ones should still reflect, since they run away after a few seconds of close proximity to someone.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/goldgrub.dm [hunk 1]
```diff
@@ -133,8 +133,11 @@
 		qdel(src)
 
 /mob/living/simple_animal/hostile/asteroid/goldgrub/bullet_act(obj/projectile/P)
-	visible_message(span_danger("The [P.name] is repelled by [name]'s girth!"))
-	return BULLET_ACT_BLOCK
+	if(istype(P, /obj/projectile/colossus))
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/goldgrub.dm:136]: I think it should only block if the goldgrub is alive instead

Comment (1393F) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/goldgrub.dm]: https://github.com/tgstation/tgstation/pull/72635/commits/78afdd3db7b10ad5af8e55fcd83bb68a5ddf680e

**Comment by newfren** [comment]
if a live one reflects the bullets, can a lazarus injector'd goldgrub be used for the same purpose?

a sentient one almost certainly can but at that point it's a significant investment of resources to kill something that doesn't even drop anything good so it's whatever

**Comment by EuclidianGeometer** [comment]
it drops voice of god, which is great.

**Pull request merged**